test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -29,7 +30,7 @@ function parent() {
 
   // Write until the buffer fills up.
   do {
-    var buf = new Buffer(BUFSIZE);
+    buf = new Buffer(BUFSIZE);
     buf.fill('.');
     sent += BUFSIZE;
   } while (child.stdin.write(buf));
```

Comment:
No, i mean renaming the two `var buf` in these loops to `buf1` and `buf2` or more descriptive names.